static/500.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
I don't think you have to have a different 500 file for nginx, you can bind the other file like here: https://github.com/WikipediaLibrary/TWLight/blob/master/docker-compose.yml#L68